src/ontology/doid-edit.owl

```diff
@@ -40307,6 +40318,88 @@ AnnotationAssertion(rdfs:label obo:DOID_0060911 "karyomegalic interstitial nephr
 SubClassOf(obo:DOID_0060911 obo:DOID_1063)
 SubClassOf(obo:DOID_0060911 ObjectSomeValuesFrom(obo:IDO_0000664 obo:GENO_0000148))
 
+# Class: obo:DOID_0060912 (craniosynostosis 7)
+
+AnnotationAssertion(obo:IAO_0000115 obo:DOID_0060912 "A synostosis that is characterized by a primary abnormality of skull growth involving premature fusion of the cranial sutures such that the growth velocity of the skull often cannot match that of the developing brain. This produces skull deformity and, in some cases, raises intracranial pressure, which must be treated promptly to avoid permanent neurodevelopmental disability
+
+Susceptibility to craniosynostosis-7 (CRS7) is conferred by heterozygous mutation in the SMAD6 gene (602931) on chromosome 15q22. Variation in SMAD6 is weakly penetrant except in the presence of a modifier, the risk allele of a common variant near the BMP2 gene (112261) on chromosome 20p12.")
+AnnotationAssertion(oboInOwl:hasDbXref obo:DOID_0060912 "OMIM:617439")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:DOID_0060912 "CRS7")
+AnnotationAssertion(oboInOwl:hasOBONamespace obo:DOID_0060912 "disease_ontology")
+AnnotationAssertion(oboInOwl:id obo:DOID_0060912 "DOID:0060912")
+AnnotationAssertion(rdfs:label obo:DOID_0060912 "craniosynostosis 7"@en)
+SubClassOf(obo:DOID_0060912 obo:DOID_2340)
+SubClassOf(obo:DOID_0060912 ObjectSomeValuesFrom(obo:IDO_0000664 obo:GENO_0000930))
+
+# Class: obo:DOID_0060913 (Usher syndrome type IIC)
+
+AnnotationAssertion(obo:IAO_0000115 obo:DOID_0060913 "A Usher syndrome characterized by  by a homozygous or compound heterozygous mutation in the ADGRV1 gene (602851) on chromosome 5q14. It is also caused by biallelic digenic mutation in the ADGRV1 and PDZD7 (612971) genes.")
+AnnotationAssertion(oboInOwl:hasDbXref obo:DOID_0060913 "OMIM:605472")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:DOID_0060913 "USH2C")
+AnnotationAssertion(oboInOwl:hasOBONamespace obo:DOID_0060913 "disease_ontology")
+AnnotationAssertion(oboInOwl:id obo:DOID_0060913 "DOID:0060913")
+AnnotationAssertion(rdfs:label obo:DOID_0060913 "Usher syndrome type IIC"@en)
+SubClassOf(obo:DOID_0060913 obo:DOID_0050439)
+SubClassOf(obo:DOID_0060913 ObjectSomeValuesFrom(obo:IDO_0000664 obo:GENO_0000930))
+
+# Class: obo:DOID_0060914 (Proteasome-associated autoinflammatory syndrome 1)
+
+AnnotationAssertion(obo:IAO_0000115 obo:DOID_0060914 "This autosomal recessive systemic autoinflammatory disorder is characterized by early childhood onset of annular erythematous plaques on the face and extremities with subsequent development of partial lipodystrophy and laboratory evidence of immune dysregulation. More variable features include recurrent fever, severe joint contractures, muscle weakness and atrophy, hepatosplenomegaly, basal ganglia calcifications, and microcytic anemia.")
```

Comment:
Modifications have been made.

Reply:
I thought this was fixed? Are there changes that have been made but not pushed?

Reply:
Need to specify what the "has_material_basis_in genetic inheritance" actually is. We may want to split this disease in two (monogenic version vs digenic version) given the different genetic etiology. Should discuss with @lschriml, as this would differ from the initial approach we've used for digenic diseases.

Reply:
Great idea. I believe making them two different diseases could be confusing to the users. Potentially, we could leave this disease entry as is but remove the genetic inheritance and create two subtypes under it that specify genetic inheritance: one monogenic and one digenic. I'll do some research on it and definitely agree on discussing with @lschriml

Reply:
I met with @lschriml to address this case. We have finalized, made some adjustments and committed the definition with a consensus of keeping it as a single disease entry while specifying the two distinct genetic inheritances.